vignettes/blueprint/actors.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Actors"
+author: "NEST CoreDev"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Actors}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+There are two main actors in the `teal` development:
+
+* `teal` App Developer
+* `teal` Module Developer
+
+## `teal` App Developer
+
+```{r, child="_setup.Rmd"}
+```
+
+```{r actors_mermaid1, echo=FALSE}
+shiny::pre(
+  class = "mermaid",
+  "
+%% This is a mermaid diagram, if you see this the plot failed to render. Sorry.
+graph LR
+    A[Teal App Developer]--utilizes--> B[teal modules]
```

Comment:
Please keep package name lowercase. We might even code-format it (haven't tested)

```suggestion
    A[teal App Developer]--utilizes--> B[teal modules]
```